cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -154,6 +154,7 @@ ExternalProject_Add(
     install
     ${boost_list}
     "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    --ignore-site-config
```

Comment:
Where should i add a comment? In url_sha1_boost.cmake.in?
Gentoo has a site config with Gentoo specific content that only the patched bjam of Gentoo can understand.
Hunter builds/uses its own bjam.

I think this Gentoo site config could be a problem for everything using bjam inside of hunter.

url_sha1_boost_library.cmake.in is generating its own .jam file. Thus the site config is not used.

Going to update the hunter.cmake files tomorrow after work.